pkg/services/entities/models.go

```diff
@@ -0,0 +1,126 @@
+package entities
+
+import (
+	"encoding/json"
+	"errors"
+	"time"
+)
+
+type EntityKind int
+
+const (
+	Panel EntityKind = iota
+	Variable
+)
+
+// Entity is the model for entity definitions.
+type Entity struct {
+	ID          int64  `xorm:"pk autoincr 'id'"`
+	OrgID       int64  `xorm:"org_id"`
+	FolderID    int64  `xorm:"folder_id"`
+	UID         string `xorm:"uid"`
+	Name        string
+	Kind        EntityKind
+	Type        string
+	Description string
```

Comment:
These 3 props perhaps need better naming? 

```suggestion
	Type int64
	InstanceType string
	InstanceDescription string
```